operator-k8s/src/crd/v1alpha1/cluster.rs

```diff
@@ -36,6 +36,9 @@ pub(crate) struct ClusterSpec {
     /// Xline container specification
     #[cfg_attr(test, garde(skip))]
     pub(crate) container: Container,
+    /// The affinity of the xline node
+    #[cfg_attr(test, garde(skip))]
+    pub(crate) affinity: Option<Affinity>,
```

Comment:
Should we add `#[serde(skip_serializing_if = "Option::is_none")]`

Reply:
fixed